### ISSUE TITLE
Fun find-fountains

### ISSUE THREAD

**Issue by eferreA9** [zzdzr/Fun#1]
Hello, I'm trying to run the Fun pipeline and I have some trubles with the last command that I have to use.

I used my own data to perform this pipeline and also I download one mcool file from mendeley database that is avialbe and I have the same trouble in both cases.

This is the command line that I used:

Fun find-fountains mcool_matrix/Repli-HiC_K562_WT_BrdU_label_release_0h.mcool::resolutions/10000 --ext_length 500000     --half_width 2 --norm VC_SQRT     --region_path output_dir_test/SoN_summits/Summits_10000_merged.bed    --extension_pixels 10 100 5     --offset 50000 --interval_length 50000  --signal_noise_background 1.3 1.3 1.3 1.3 1.3   --coverage_ratio 0 --p_value 0.05    --output output_dir/fountains_10kb

And I get this error:

TypeError: ufunc 'floor_divide' not supported for the input types, and the inputs could not be safely coerced to any supported types according to the casting rule ''safe''

Then I find that there was a problem with the bed file and I remove header line from the bed file (the first line) and now I have this error:

 File "/usr/local/ohpc/pub/apps/Fun/1.0.1/lib/ks_test.py", line 87, in <listcomp>
    i, scipy.stats._stats_py.KstestResult
AttributeError: module 'scipy.stats' has no attribute '_stats_py'

I don't understand what may happen here, If someone has an idea of how to solve it I would be very thankfull.

Thanks.

**Comment by zzdzr** [comment]
Hey, It appears that "_stats_py" is not in your scipy.stats, please confirm your version of scipy, and if it doesn't work, I will go check my program again.

**Comment by eferreA9** [comment]
Hi! My scipy version is 1.6.3, I don't know if I need another version and if its the case which one.

**Comment by zzdzr** [comment]
My current version is 1.9.1, maybe you can try to update it, please let me know if you fix it.

**Comment by eferreA9** [comment]
I tried to run the command line with this version, and it didn't work. What Python version do you use? I have 3.8. Also, could you provide me with the requirements.txt file to see the versions of all the modules in Python? Thanks.

**Comment by zzdzr** [comment]
Of course, you can email me a screenshot of the specific error message, and I will help you resolve the issue in my spare time.

**Comment by eferreA9** [comment]
![image](https://github.com/zzdzr/Fun/assets/157719630/495ba678-3ea4-479e-90bd-f075ca658921)
 this is all the error, so do you have a reqyurements.txt?

**Comment by eferreA9** [comment]
This is the screenshot of the specific error, so do you have the requirements.txt?

![image](https://github.com/zzdzr/Fun/assets/157719630/8fee9def-3c01-449c-b6c8-b8fbd142778f)

**Comment by zzdzr** [comment]
I will test the code and then provide the versions of the libraries used in the environment. I will notify you in the next few days if I manage to pinpoint the issue.

**Comment by eferreA9** [comment]
Okay! Thanks a lot, I'm looking forward to hearing back from you with the library versions used.

**Comment by zzdzr** [comment]
Apologies for the wait. After testing, my current environment is functioning normally. The main dependency versions I am using are as follows:

numba: 0.56.2
pandas: 1.4.1
numpy: 1.23.3
bioframe: 0.3.3
scipy: 1.9.1
cooler: 0.8.11
click: 7.1.2

I noticed that when you execute the Fun command, you do not specify the version of Python. Is it possible that the Python version you installed the dependencies with is different from the version you are using to execute Fun? To avoid this problem, perhaps you can specify the Python version, for example, by using python3.8 ./Fun find-fountains.
